config/prod/wpoehlm-parallel-ec2-scicomp.yaml

```diff
@@ -0,0 +1,28 @@
+# Provision an EC2 instance
+template_path: managed-ec2-v6.yaml
```

Comment:
there's a new version now.  please refer to the latest template, https://github.com/Sage-Bionetworks/scicomp-provisioner/pull/3/files